rfc.xml

```diff
@@ -319,52 +317,59 @@
 		</section>
 		<section anchor="formats" title="Data Element Formats">
 			<t>
-				The following are definitions of the data types we will be
-				creating to support this new feature set.
+				The following are format specifications for the keylist file and
+				its signature file.
 			</t>
 			<section anchor="keylist-format" title="Keylist">
 				<t>
-					The keylist MUST be formatted using JavaScript Object Notation
+					The keylist MUST be a valid JavaScript Object Notation
 					(JSON) Data Interchange Format
-					<xref target="RFC8259" />
-					with specific required keys and values to be defined.
+					<xref target="RFC8259" /> object with specific keys
+					and values, as defined below. Note that unless otherwise specified,
+					'key' in this section refers to JSON keys--not OpenPGP keys.
+				</t>
+				<t>
+					To encode metadata, the keylist MUST have a "metadata" root key
+					with an object as the value ("metadata object").
+					The metadata object MUST contain a "signature_uri" key whose value
+					is the URI string of the keylist's signature file. All metadata keys 
+					apart from "signature_uri" are OPTIONAL.
+				</t>
+				<t>
+					The metadata object MAY contain a "keyserver" key with the value of the
+					URI string of the keyserver from which the OpenPGP keys in the keylist
+					should be retrieved.
 				</t>
 				<t>
-					The keylist MUST have a "metadata" key with an object as the
```

Comment:
Note that I deleted this paragraph, but merged it into the above—despite what the diff shows, it's not all gone!